Milestone 1

Haywhizzz opened the pull request:
In this pull request I have:

- Added prop-types using npm for type checking props
- Added big.js using npm

Haywhizzz committed:
Add react app

Haywhizzz committed:
Add prop-types

Haywhizzz committed:
Install big.js

Haywhizzz committed:
Update Readme

Haywhizzz committed:
Fix eslint errors

Haywhizzz committed:
Fix stylelint errors

Haywhizzz committed:
Fix linter errors

Haywhizzz committed:
Fix linters

trekab reviewed (changes requested):
#### Changes Required! :recycle: 
Hi @Haywhizzz! Well done adding the necessary packages as required on this milestone. There are a few changes for you to work on to complete it though. These include:
- [x] Delete the unnecessary files or assets, such as the React svg Logo. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/files-removed.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/))
- [x] Deployed your app on Heroku(or other hosts like Netlifly, Firebase, etc) and add the live link to your README file.([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/app-deployed.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/))
- [x] Please customize your README.md file a bit further. It actually has no relevant information about your project other than the author's details. Feel free to use [this template](https://github.com/microverseinc/readme-template)

__Please make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__[Treasure Kabareebe](https://trekab.github.io)__

Haywhizzz committed:
Update Readme

Haywhizzz committed:
Update Readme

Haywhizzz committed:
Update README.md

Haywhizzz committed:
Delete unnecessary file

Haywhizzz committed:
Merge branch 'milestone-1' of https://github.com/Haywhizzz/React-Calculator into milestone-1

Haywhizzz committed:
Remove unnecessary file

Haywhizzz committed:
Remove extra files created by react

Haywhizzz committed:
Fix eslint errors

Haywhizzz committed:
Fix eslint error

Haywhizzz commented:
I have removed the unnecessary file, deployed the project, and also customized the readme.

> #### Changes Required! ♻️
> Hi @Haywhizzz! Well done adding the necessary packages as required on this milestone. There are a few changes for you to work on to complete it though. These include:
> 
> * [x]  Delete the unnecessary files or assets, such as the React svg Logo. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/files-removed.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/))
> * [x]  Deployed your app on Heroku(or other hosts like Netlifly, Firebase, etc) and add the live link to your README file.([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/app-deployed.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/))
> * [x]  Please customize your README.md file a bit further. It actually has no relevant information about your project other than the author's details. Feel free to use [this template](https://github.com/microverseinc/readme-template)
> 
> **Please make the necessary changes and submit another code review request.**
> _Feel free to reach me on slack for any inquiries about this review._ 😉
> **[Treasure Kabareebe](https://trekab.github.io)**

I have removed the unnecessary file, deployed the project, and also customized the readme.

Forison reviewed (approved):
## Status::Approved✔️

# 👋 @Haywhizzz 

### Good job on addressing the issues raised in the previous review. kindly go ahead and merge🙌 

### See you in the next milestone💪

#### Happy coding

Addo Forison
